nav2_bringup/launch/nav2_bringup_launch.py

```diff
@@ -14,118 +14,85 @@
 
 import os
 
-from ament_index_python.packages import get_package_prefix
-from ament_index_python.packages import get_package_share_directory
-from launch.conditions import IfCondition
-
-import launch.actions
-import launch_ros.actions
+from ament_index_python.packages import get_package_prefix, get_package_share_directory
 
-from nav2_common.launch import RewrittenYaml
+from launch import LaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.conditions import IfCondition
```

Comment:
Is 'IfCondition' used in this file? I don't see it.

Reply:
It's not, removed it.